components/Card.js

```diff
@@ -0,0 +1,79 @@
+import PropTypes from 'prop-types';
+import React from 'react';
+
+function Card({ backgroundImageSource, children }) {
+  return (
+    <div className="card shadow-xl image-full">
+      {backgroundImageSource && (
+        <figure>
+          <img src={backgroundImageSource} />
+        </figure>
+      )}
+      <div className="card-body justify-end space-y-3">{children}</div>
+    </div>
+  );
+}
+
+Card.propTypes = {
+  backgroundImageSource: PropTypes.string,
+  children: PropTypes.node.isRequired,
+};
+
+export function CardTitle({ children }) {
+  return <span className="card-title m-0">{children}</span>;
+}
+
+CardTitle.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export function CardParagraph({ children }) {
+  return <span>{children}</span>;
+}
+
+CardParagraph.propTypes = {
+  children: PropTypes.node.isRequired,
+};
+
+export function CardAction({ children, linkTo, isPending, onClick }) {
+  return (
+    <div className="pt-5">
+      {linkTo ? (
+        <a className="btn btn-primary" href={linkTo}>
+          {children}
+        </a>
+      ) : (
+        <button
+          className={`btn btn-primary ${isPending ? 'loading' : ''}`}
```

Comment:
Thanks for your extra eye for detail here, tracking `isPending` state with the `loading` indicator. This sort of detail touch really matters and makes the experience better.